io.weldr.cockpit-composer.metainfo.xml

```diff
@@ -0,0 +1,15 @@
+<component type="addon">
+  <id>io.weldr.welder-web.metainfo.xml</id>
```

Comment:
` io.weldr.cockpit-composer`?

Reply:
But I am confused about the need to duplicate things, so maybe `welder-web` as the id is correct.

Reply:
@mvollmer: It's still the upstream github project name, and https://weldr.io its upstream homepage. Is the ID visible to users anywhere? I thought it was just an internal disambiguation for the API?

Reply:
We do want to change the name to cockpit-composer everywhere. I haven't done so yet, because I'm afraid I'll miss and break something. I think we should go with cockpit-composer here already. The change is coming in the near future.

Reply:
Could we have just a single file instead of two files with identical content?  That would be less confusing, I think.

Reply:
It's not identical content, the `<Name>` is different. And yes, the duplication is confusing, but as long as we have it, we should be consistently doing it. We can also block this PR on doing the un-duplication, if that happens soon (or I need to carve out some time to do it)